proposals/simd/SIMD.md

```diff
@@ -685,12 +698,32 @@ Load a single element and splat to all lanes of a `v128` vector.
 
 Fetch consequtive integers up to 32-bit wide and produce a vector with lanes up to 64 bits.
 
+```python
+def S.load_extend(ext, memarg):
+    result = S.New()
+    bytes = memory.bytes[memarg.start:memarg.start + 8])
+    for i in range(S.Lanes):
+        result[i] = ext(S.LaneType.from_byte(bytes[(i * S.LaneBytes/2):((i+1) * S.LaneBytes/2)]))
+    return result
+
+def S.load_extend_s(memarg):
+    return S.load_extend(Sext, memarg)
+
+def S.load_extend_u(memarg):
+    return S.load_extend(Zext, memarg)
+```
+
 ### Store
 
 * `v128.store(memarg, data: v128)`
 
 Store a `v128` vector to the given heap address.
 
+```python
+def S.store(memarg, a):
+    memory.bytes[memarg.start:memarg.start + 16] = S.bytes
```

Comment:
We might want to define what `S.bytes` mean?

Reply:
I guess we might need a better way to describe that, since `S` seems to be an array of lane values in other cases.

Reply:
`bytes()` seems to be a the Python function to convert something to a byte array, that is probably better than `S.bytes`